src/main/java/winter/Manager.java

```diff
@@ -0,0 +1,302 @@
+            }
+
+            switch (action) {
+            case TODO:
+            case DEADLINE:
+            case EVENT:
+                addTask(action,inputString.substring(spaceIndex,slashIndex),inputString,slashIndex);
+                continue;
+            case BYE:
+                sayBye();
+                flag = false;
+                break;
+            case LIST:
+                displayList(taskList, taskIndex);
+                continue;
+            case MARK:
+                taskList[Integer.parseInt(inputString.substring(5)) - 1].mark();
+                continue;
+            case UNMARK:
+                taskList[Integer.parseInt(inputString.substring(7)) - 1].unmark();
+                continue;
+            case INVALIDTODO:
+            case INVALIDDEADLINE:
+            case INVALIDEVENT:
+            case INVALIDCOMMAND:
+            case EMPTYCOMMAND:
+                handleCheckedExceptions(action);
+
+            }
+
+        } while (flag);
+
+    }
+    private static void addTask(Commands taskType, String taskName, String inputString, int slashIndex) {
+        ToDo newToDo = null;
+        Deadline newDeadline = null;
+        Event newEvent = null;
+        switch (taskType) {
+        case TODO:
+            newToDo = new ToDo(taskIndex, false, taskName);
+            taskList[taskIndex] = newToDo;
+            taskIndex++;
+            break;
+        case DEADLINE:
+            String deadline = inputString.substring(slashIndex+3);
+            newDeadline = new Deadline(taskIndex,false,taskName,deadline);
+            taskList[taskIndex] = newDeadline;
+            taskIndex++;
+
+            break;
+        case EVENT:
+            String startAndEnd = inputString.substring(slashIndex+5);
+            String startTime = startAndEnd.substring(0,startAndEnd.indexOf("/"));
+            String endTime = startAndEnd.substring(startAndEnd.indexOf("/")+3);
+            newEvent = new Event(taskIndex,false,taskName,startTime,endTime);
+            taskList[taskIndex] = newEvent;
+            taskIndex++;
+
+            break;
+        }
+
+
+        System.out.print(line);
+        System.out.print(indent);
+        System.out.println("OK, I've added: " + taskName);
+        if (taskList[taskIndex-1].getType().equals("D")) {
+            System.out.println(newDeadline);
+
+        }else if (taskList[taskIndex-1].getType().equals("E")){
+            System.out.println(newEvent);
+
+        }else {
+            System.out.println(newToDo);
+        }
+
+        System.out.println(indent + "Now, you have " + taskIndex+ " tasks in your list.");
+        System.out.print(line);
+    }
+    private static Commands classifyCommand(String inputString) {
+        switch (inputString) {
+        // Cases include farewell and list commands
+        case "bye":
+        case "Bye":
+        case "BYE":
+            return BYE;
+        case "list":
+        case "List":
+        case "LIST":
```

Comment:
Understand what you are trying to do here, though maybe using `"some_string". toLowerCase()` can help you normalise the command word so you would not have to account for that many edge cases? This is outside of code quality / coding standard, just a tip that is hopefully helpful